web-ui/src/main/resources/catalog/templates/top-toolbar.html

```diff
@@ -63,8 +63,8 @@
            title="{{'editorBoard' | translate}}"
            class="dropdown-toggle gn-menuheader-xs"
            role="button" aria-expanded="false">
-          <i class="fa fa-fw fa-pencil hidden-sm"></i>
-          <span translate>contribute</span>
+          <i class="fa fa-fw fa-pencil"></i>
+          <span translate class="hidden-md hidden-sm">contribute</span>
```

Comment:
I noticed that you added the class "hidden-md hidden-sm" but you did not add the same for the globe and search.  It seems to be causing some inconsistencies in the menu. Where some are converted to icons but the others are not?
![image](https://user-images.githubusercontent.com/1868233/98119599-32fe2e80-1e83-11eb-851a-7c48669523fc.png)

Was this done on purpose?